app/views/hub/organizations/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<% @title = t(".title") %>
+<% content_for :page_title, @title %>
+<% content_for :card do %>
+  <div class="hub-form">
+    <%= form_with model: @vita_partner, url: hub_organizations_path, method: :post, local: true, builder: VitaMinFormBuilder do |f| %>
```

Comment:
These forms always confuse me ... I was wondering how the model: @vita_partner worked (normally im needing to do [:hub, @vita_partner]), but then i realized that you've got an explicit url + method which overrides it. makes sense

Reply:
Oh interesting! Is it clearer if we remove the explicit `url:` here? Earnestly curious.

Reply:
Consider calling this `@organization`